matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/database/RealmSessionStoreMigration.kt

```diff
@@ -381,6 +383,13 @@ internal class RealmSessionStoreMigration @Inject constructor(
 
     private fun migrateTo19(realm: DynamicRealm) {
         Timber.d("Step 18 -> 19")
+        realm.schema.get("DraftEntity")
+                ?.addField(DraftEntityFields.MESSAGE_TYPE, String::class.java)
+                ?.setRequired(DraftEntityFields.MESSAGE_TYPE, true)
+                ?.transform {
+                    it.setString(DraftEntityFields.MESSAGE_TYPE, MessageType.MSGTYPE_TEXT)
+                }
+
```

Comment:
This will be a problem for those who are already on version 19. In this case you should create a version 20.